docs/vexhub.md

```diff
@@ -0,0 +1,59 @@
+# VEX
+
+VEX is a format used to convey information about the exploitability of vulnerabilities in software products and share them with scanning tools.
+
+The use of this format can sensitively reduce the number of vulnerabilities in the final vulnerability report, which is more often full of false positives entries.
+
+If you want to know more about VEX, take a look [here](https://github.com/openvex/spec).
+
+## VEX Hub
+
+VEX Hub is a place where VEX (Vulnerability Exploitability eXchange) documents from different open-source projects are collected and organized. It helps people and security tools find and use important information about software vulnerabilities more easily.
+
+## Getting Started
+
+In order to scan `Registries` using VEX, you just have to create (one, or more) `VEXHub` resources.
+
+Here's an example of `VEXHub` resource (you can find it under [`examples/vexhub.yaml`](https://github.com/rancher-sandbox/sbombastic/blob/main/examples/vexhub.yaml):
+
+```yaml
+apiVersion: sbombastic.rancher.io/v1alpha1
+kind: VEXHub
+metadata:
+  name: company-vexhub
+spec:
+  url: "https://github.com/rancher/vexhub"
+  enabled: true
+```
+
+Apply the resource with:
+
+```sh
```

Comment:
I think you should either use `console` or `bash`. I don't think `sh` is recognized (at least not by docusaurus)